### DISCUSSION TITLE
Build not the whole documentation

### DISCUSSION THREAD

**Discussion by Vetrenik** [squidfunk/mkdocs-material#5256] [Q&A] [answered]
Hello, everyone!

Is there any way to build not the whole documentation into static site? The case is - we have different customers, which bought different licenses of the product and we want to provide them with documentation in accordance to their licenses.

**Comment by squidfunk** [comment, marked as answer]
You could divide your documentation into different projects and then use one of the multi-project plugins (monorepo, multirepo) to build a consolidated site. See the [MkDocs plugin wiki](https://github.com/mkdocs/mkdocs/wiki/MkDocs-Plugins#site-management) for more information.

**Comment by Vetrenik** [comment]
Thx a lot, will research in that way

сб, 25 мар. 2023 г., 14:55 Martin Donath ***@***.***>:
